vowpalwabbit/cb_adf.cc

```diff
@@ -399,7 +399,7 @@ void finish_multiline_example(vw& all, cb_adf& data, multi_ex& ec_seq)
     output_example_seq(all, data, ec_seq);
     global_print_newline(all);
   }
-  VW::clear_seq_and_finish_examples(all, ec_seq);
```

Comment:
Why do we no longer want to clear the sequence?

Reply:
It is moved to learner.cc::multi_example_handler.
clear_seq_and_finish_example did 2 things:
- finish examples (only if example is from certain vw instance's ring)
- clear sequence without any condition.
So,
1) in general, it is potential memory leak (if ec_seq is the only reference to non-ring examples).
2) particularly for multiinstance execution, it breaks the trick with using instance that allocated example as the last one.

Reply:
Also, ec_seq is std vector, so there is no memory management reason for doing explicit clean. We are doing it just because  the same ec_seq instance is used for every multi_line_example. But in this case it is purely piece of driver's business logic.